### DISCUSSION TITLE
Are there any documentation for applying inverse text normalization in another language?

### DISCUSSION THREAD

**Discussion by tserdar** [NVIDIA-NeMo/NeMo#4422] [Q&A] [answered]
I am currently looking for a way to apply a state-of-the-art inverse text normalization method on Danish. 

I installed and tested NeMo for English and it works quite well. In the paper it is also mentioned that it is possible to apply the idea to a different language. 

But I cannot see any documentation/guide to be able to do so. Could you please help me out?

**Comment by ekmb** [comment, marked as answer]
Documentation and tutorials could be found [here](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/nlp/text_normalization/intro.html).
